.github/workflows/ci_tests.yml

```diff
@@ -28,11 +28,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip wheel
         pip install -e .
+        # uninstall, otherwise git+... installation from .ci_pip_reqs.txt won't do anything
         pip uninstall -y petab
         pip install -r .ci_pip_reqs.txt
```

Comment:
Does this work?
```suggestion
        pip install -r .ci_pip_reqs.txt
        pip install -e .
```

Reply:
No, I changed the order because we need .ci_pip_reqs.txt afterwards to update sympy, which will otherwise be downgraded by pysb, which unfortunately pins sympy to < 1.12

Reply:
I see, and you probably want to avoid `--force-reinstall` for all. Then :+1: